File: JavaScript/58. Length of Last Word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    var count = 0;
    var n = s.length - 1;
    while (n >= 0 && s[n] == ' ') {
        n--;
    }
    for (var j = n; j>=0;j--) {
        if(s[j]==' '){
            return count;
        }
        count++;
    }
    return count;
};

File: C++/58. Length of Last Word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count = 0;
        int n = s.size() - 1;
        while (n >= 0 && s[n] == ' ') {
            n--;
        }
        for (int j = n; j>=0;j--) {
            if(s[j]==' '){
                return count;
            }
            count++;
        }
        return count;
    }
};
